frontend/libs/studio-components/src/components/StudioContentMenu/StudioMenuTab/StudioMenuTabContainer.module.css

```diff
@@ -0,0 +1,39 @@
+:root {
+  --selected-border-marking: 3px;
```

Comment:
Should this be a design token? 😄

Reply:
I cant find any fds design token that is `3px` can we use a spacing token with `rem` here @Annikenkbrathen ?
Or one of these?
```
--fds-border_radius-interactive: 4px;
--fds-border_radius-small: 2px;
--fds-border_radius-medium: 4px;
```

Reply:
Don't thin they have 3px, but what about using the one with 4px? 😄